packages/cli-core/src/format/format.ts

```diff
@@ -71,6 +80,18 @@ export class Format {
     Object.entries(record)
       .map(([key, value]) => `${key}: ${String(value)}`)
       .join(os.EOL);
+  color = (message: string, color: Color) => color(message);
 }
 
+export type Color = Colorize;
+export type ColorName = keyof typeof colors;
+
+export const colors = {
```

Comment:
this is still exporting kleur functions. Why not have
```ts
export enum Color {
  GREEN = 'green';
  YELLOW = 'yellow';
  etc
}
```

Then internally map those enum values to the kleur function.

Reply:
Updated to not export the `colors` object but just a type and enumerated color names.